1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,79 @@
-// Write your code here
+const images = [
```

Comment:
Great work again Laleh!!!  🙌  🎉  🦦 

Again, nothing much to say here, it's all really really good code!!! You should be super proud! 👩‍💻  💃  🥇 

The only thing I would suggest you work on now is your CSS and design skills. Maybe try center aligning the button bar to the images and making them look nice. I added this to the css
```
body {
	background-color: bisque;
}

.container {
	display: flex;
	align-items: center;
	justify-content: center;
	flex-direction: column;
}

button {
	background-color: rgb(36, 157, 25);
	color: white;
	border: 0 solid white;
	padding: 20px;
	margin: 5px;
	width: 180px;
	height: 0;
	border-radius: 5px;
	height: 100%;
	font-weight: 700;
}
```
so it looked like this 
<img width="1237" alt="image" src="https://user-images.githubusercontent.com/50834877/154805573-cdc27ae2-87c5-42ee-ac42-a45c3b311096.png">

Certainly nothing amazing, but much better than just boring standard HTML.

Again, great work and keep it up!!! 🎉